Sat Apr 12 11:30:59 CEST 2008  Sjoerd Simons <sjoerd@luon.net>
  * When removing a media stream before the initiate is sent, send it if possible
diff -rN -u old-telepathy-gabble/src/gabble-media-session.c new-telepathy-gabble/src/gabble-media-session.c
--- old-telepathy-gabble/src/gabble-media-session.c	2008-04-12 17:45:38.000000000 +0200
+++ new-telepathy-gabble/src/gabble-media-session.c	2008-04-12 17:45:38.000000000 +0200
@@ -2320,6 +2320,10 @@
       GMS_DEBUG_INFO (session, "not sending jingle session action "
           "\"content-remove\" to peer, no initiates or adds sent for "
           "these streams");
+      if (priv->state < JS_STATE_PENDING_INITIATE_SENT)
+        {
+           try_session_initiate (session);
+        }
     }
 }
 

